### PR TITLE
implement process 2019 + 1-month transition period

### PR DESCRIPTION
**Pull request by deniak**

### CODE REVIEW

File: test/docs/sotd/deprecated2018-allowed.html [hunk 1]
```diff
@@ -8,10 +8,10 @@
   <body>
     <div class="head">
       <h1>Simple baseline headers valid document</h1>
-      <h2>W3C Working Draft 15 February 2018</h2>
+      <h2>W3C Working Draft 15 March 2019</h2>
     </div>
     <h2>Status of This Document</h2>
-    <p>This document is governed by the <a id="w3c_process_revision" href="https://www.w3.org/2017/Process-20170301/">1 March 2017 <abbr title="World Wide Web Consortium">W3C</abbr> Process Document</a>.</p>
+    <p>This document is governed by the <a id="w3c_process_revision" href="https://www.w3.org/2018/Process-20190201/">1 February 2018 <abbr title="World Wide Web Consortium">W3C</abbr> Process Document</a>.</p>
```
Comment (plehegar) [inline test/docs/sotd/deprecated2018-allowed.html]: should read Process-20180201

Comment (deniak) [reply test/docs/sotd/deprecated2018-allowed.html]: Good catch!

**Comment by plehegar** [comment]
This is for https://github.com/w3c/modern-tooling/issues/84

**Comment by deniak** [comment]
Will deploy tomorrow

**Comment by deniak** [comment]
PR has been deployed and Echidna updated with this new release

**Comment by plehegar** [comment]
https://www.w3.org/pubrules/doc/rules/?profile=REC still says 2018...

**Comment by deniak** [comment]
> https://www.w3.org/pubrules/doc/rules/?profile=REC still says 2018...

Fix in https://github.com/w3c/specberus/pull/818/commits/b89565748cafb33532b2e46cb901fe62588e13ec

**Pull request merged**